### PR TITLE
topology2: add be pipeline for sdw

### PR DESCRIPTION
**Pull request by bardliao**
Use two separated pipelines for BE and FE.

Signed-off-by: Bard Liao <yung-chuan.liao@linux.intel.com>

### CODE REVIEW

File: tools/topology/topology2/cavs/cavs-sdw.conf [hunk 1]
```diff
@@ -85,38 +54,6 @@ Object.Dai {
 		Object.Base.hw_config."ALH3" {
 			id	1
 		}
-
```
Comment (plbossart) [inline tools/topology/topology2/cavs/cavs-sdw.conf:88]: line 50: why is the default_hw_confg_id '1'?

Comment (ranj063) [reply tools/topology/topology2/cavs/cavs-sdw.conf:88]: because the're sonly one hw_config and its ID is 1 on line 55

**Review by ranj063 — APPROVED** [review]

File: tools/topology/topology2/cavs/cavs-sdw.conf (hunk 1 above)
Comment (ranj063) [reply tools/topology/topology2/cavs/cavs-sdw.conf:88]: @plbossart do you mean other topologies? We blindly set the hw_config ID to match the link ID in the machine driver for no good reason.

Comment (lgirdwood) [reply tools/topology/topology2/cavs/cavs-sdw.conf:88]: We should fix this and make it non ambiguous prior to roll out of topology2. If it not needed we can remove or we should start IDs at 0. We have time to align, @ranj063 are you OK to look at this when convenient ?

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**